### PR TITLE
refactor archive, simplify

### PR DESCRIPTION
**Pull request by karissa**

### CODE REVIEW

File: client/js/models/archive.js [hunk 1]
```diff
@@ -276,9 +241,7 @@ module.exports = {
         promises.push(promise)
       } else { // download entire archive as zip
         zipName = state.key
-        var keys = Object.keys(state.entries.sort())
-        keys.forEach((key, i) => {
-          const entry = state.entries[key]
+        state.entries.forEach((entry) => {
```
Comment (laurengarcia) [inline client/js/models/archive.js]: didn't @clkao have a reason he was needing to `.sort()` these for jszip? i can't remember what the reason was tho.

Comment (karissa) [inline client/js/models/archive.js]: there wasn't a reason except it made for easier testing but we aren't testing.

Also there are no keys anymore, it's an array now so this doesn't do anything

Comment (karissa) [inline client/js/models/archive.js]: aren't testing zip download*

Comment (laurengarcia) [inline client/js/models/archive.js]: k

**Pull request merged**